composer/loggers/logger_hparams.py

```diff
@@ -82,6 +82,9 @@ class WandBLoggerBackendHparams(BaseLoggerBackendHparams):
         name (str, optional): Weights and Biases run name.
         entity (str, optional): Weights and Biases entity name.
         tags (str, optional): Comma-seperated list of tags to add to the run.
+        log_artifacts (bool, optional): Whether to log artifacts. Defaults to False.
```

Comment:
This should say "automatically log artifacts in runs folder" correct? manual logs would still be enabled?

Reply:
Theoretically, yes, but there is no api available in our trainer to call `wandb.log_artifact` (since our base logger doesn't expose an artifact API)